Cargo.toml

```diff
@@ -38,7 +38,7 @@ fuchsia-zircon = "0.3.2"
 fuchsia-zircon-sys = "0.3.2"
 
 [target.'cfg(unix)'.dependencies]
-libc   = "0.2.42"
+libc   = "0.2.54"
```

Comment:
@pfmooney I assume this is a sufficient up-to-date version, I didn't manually check this.